File: Week 1/Problem Set 1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    // Prompt the user about the height of the pyramids
    do
    {
        height = get_int("Height:");
    } while (height<1 || height>8);

    // Loop to print out the pyramids
    for (int i = 1; i <= height; i++)
    {
        // Print out left pyramid
        for (int j = 1; j <= height - i; j++)
        {
            printf(" ");
        }
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }

        // Print out middle between pyramids, which is only 1 space
        printf("  ");

        // Print out right pyramid
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Week 6/Problem Set 6/sentimental-mario-more/mario.py
height = 0

# Prompt the user about the height of the pyramids
while height < 1 or height > 8:
    try:
        height = int(input("Height: "))
    except ValueError:
        print("Invalid input. Please enter a valid height (integer).")

# Loop to print out the pyramids
for i in range(1, height + 1):
    # Print out left pyramid
    for j in range(1, height - i + 1):
        print(" ", end="")
    for j in range(1, i+1):
        print("#", end="")

    # Print out middle between pyramids, which is only 1 space
    print("  ", end="")

    # Print out right pyramid
    for j in range(1, i + 1):
        print("#", end="")
    print()